### PR TITLE
add-annotation-for-prommy

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/2587 and PM https://github.com/giantswarm/giantswarm/issues/2645

Adding this annotation will  enable monitoring all the worker endpoints

### CODE REVIEW

File: service/kvmconfig/v6/resource/service/master_service.go [hunk 1]
```diff
@@ -23,6 +23,10 @@ func newMasterService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 			},
 			Annotations: map[string]string{
 				"giantswarm.io/prometheus-cluster": key.ClusterID(customObject),
+				"prometheus.io/path":               "/healthz",
+				"prometheus.io/port":               "'30010'",
```
Comment (JosephSalisbury) [inline service/kvmconfig/v6/resource/service/master_service.go]: Why the double quotes?

File: service/kvmconfig/v6/resource/service/worker_service.go [hunk 2]
```diff
@@ -22,6 +22,12 @@ func newWorkerService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 				"customer": key.ClusterCustomer(customObject),
 				"app":      key.WorkerID,
 			},
+			Annotations: map[string]string{
+				"prometheus.io/path":   "/healthz",
+				"prometheus.io/port":   "'30010'",
```
Comment (JosephSalisbury) [inline service/kvmconfig/v6/resource/service/worker_service.go]: Same here, we want just `"30010"`

**Comment by calvix** [comment]
ping @JosephSalisbury  fixed in commit `single double quotes it is !`

**Comment by MarcelMue** [comment]
Dumb question: why are we not using https here? Looks good otherwise.

**Comment by JosephSalisbury** [comment]
Largely because we don't need to - this is pretty much an uptime check, we're not getting any metrics from the ingress controller.

**Comment by calvix** [comment]
yeah we care if TCP is avaiable that is all

File: service/kvmconfig/v6/resource/service/master_service.go [hunk 3]
```diff
@@ -23,6 +23,10 @@ func newMasterService(customObject v1alpha1.KVMConfig) *apiv1.Service {
 			},
 			Annotations: map[string]string{
 				"giantswarm.io/prometheus-cluster": key.ClusterID(customObject),
+				"prometheus.io/path":               "/healthz",
+				"prometheus.io/port":               "30010",
```
Comment (r7vme) [inline service/kvmconfig/v6/resource/service/master_service.go]: 443 ???

Comment (r7vme) [reply service/kvmconfig/v6/resource/service/master_service.go]: please ignore my comment. all good.

Comment (calvix) [reply service/kvmconfig/v6/resource/service/master_service.go]: we don't wanna scrape api here, that is already taken care off by `prometheus-config-controller`
We discussed this with @JosephSalisbury and it's still beneficial for us to monitor master endpoint ip like this.

**Pull request merged**